web/client/components/map/leaflet/plugins/index.js

```diff
@@ -1,19 +1,19 @@
-/**
```

Comment:
Do not reformat code (probably editor settings for newlines or tabs/spaces). We have and .editorconfig setting that should be usable by your editor (maybe with some plugin)